package-lock.json

```diff
@@ -0,0 +1,13221 @@
+{
```

Comment:
Please revert this file by running the following command:

```shell
git checkout upstream/develop package-lock.json
```

From the oppia directory.